test/common/test_point_type_construction.cpp

```diff
@@ -44,12 +44,68 @@
 using namespace pcl;
 using namespace pcl::test;
 
-TEST (PointTypeConstruction, PointXYZConstruction)
+static constexpr const std::uint8_t uint8_0{};
+static constexpr const std::uint8_t uint8_255{};
+static constexpr const float float_0{};
+static constexpr const float float_1{1.0f};
```

Comment:
Will `0f` and `1.f` not work directly inside the static_assert statements?

Reply:
yes, that was ugly and useless anyways :)